.gitignore

```diff
@@ -2,5 +2,5 @@ __pycache__/
 *.py[cod]
 .idea/
 *.egg*
-data/
-models/
+/data/
+/models/
```

Comment:
Why do we need to ignore these paths?

Reply:
Because data and model are the destination folder for the dataset and models downloads. I decide not to push them because the size is about 3/4 GB

Reply:
Ok, let's setup DVC pipelines and storage later.
Still, I think that the previous version was correct and that this update is wrong.

/data is an absolute path from the root of your (Linux?) machine
data/ refers to the data path relative to the .gitignore file

Reply:
With the previous change git does not track any changes in src/models and src/data but now the only folders ignored are the ones in the root folder. So in windows machine i think it is the proper way to ignored these folders.

Reply:
@gianfrancodemarco you can test on you Linux machine and we can figure out the right solution

Reply:
Why would it ignore folders under src/?

Reply:
Both seems to be working, because the leading / referres to the project root. Choose the one that you prefere

Reply:
i try with the old version and now works. I revert this commit. Sry